poly-commitment/src/lib.rs

```diff
@@ -83,6 +83,10 @@ pub trait SRS<G: CommitmentCurve> {
         plnm: &Evaluations<G::ScalarField, D<G::ScalarField>>,
         rng: &mut (impl RngCore + CryptoRng),
     ) -> BlindedCommitment<G>;
+    ///for now needed by snarky-rs
```

Comment:
Would be nice to add more documentation. It is not enough to say "for now needed by snarky-rs". For which reason it is needed?

Reply:
Why not exposing simply create_trusted_setup?

Reply:
Let's do it in a follow-up. We should refactorize the SRS structure/type/traits. We should have different type of SRS. One when there is a toxic waste, one when it is transparent.

Reply:
See https://github.com/o1-labs/proof-systems/issues/1721